torch/fx/experimental/proxy_tensor.py

```diff
@@ -93,10 +89,10 @@ def maybe_disable_fake_tensor_mode():
         return nullcontext()
 
 
-def unwrap_elem(e):
-    if isinstance(e, ProxyTensor):
-        return e.elem
-    return e
+@dataclass
+class _ProxyTensor:
+    proxy: Proxy
+    constant: Optional[torch.Tensor]
```

Comment:
scary to me memory wise to be carrying around a tensor... when will those be set ?

Reply:
These only are ever set for torch.tensor(...) constructor call, and any subsequent operations that only have constants. So it is very, very limited. Factory functions don't count as constants.

Reply:
Yes, that's right. We can modify the policy to check the numel of the result and bail if it is bigger than one. This is a preexisting problem though, we should hit it in a separate PR